cmd/k8s-operator/proxy.go

```diff
@@ -36,12 +37,6 @@ var whoIsKey = ctxkey.New("", (*apitype.WhoIsResponse)(nil))
 var (
 	// counterNumRequestsproxies counts the number of API server requests proxied via this proxy.
 	counterNumRequestsProxied = clientmetric.NewCounter("k8s_auth_proxy_requests_proxied")
-
```

Comment:
I have moved these into the new sessionrecordig package (as they also need to be accessed by the hijacker who lives there)